src/process.c

```diff
@@ -226,9 +226,7 @@ cc_oci_setup_shim (struct cc_oci_config *config,
 	ret = true;
 
 out:
-	if (tty_fd > 2) {
```

Comment:
Indeed, but I don't think that is correct behaviour. I've just updated my test which should guarantee the fd gets closed in all scenarios.

@amshinde - could you review this?